### ISSUE TITLE
Request length overflow

### ISSUE THREAD

**Issue by Dil3mm4** [LorenzHW/My-Telegrams#6]
```
Invalid speech in response
Telegram Connect
Request Identifier: null

Speech with id amzn1.echo-api.request.************************************ and length 9503 exceeds allowed size of 8000 characters
```

A truncate option should be considered to either send many speech responses as required (length divided per size) or just catch the exception truncating the whole thing to 8000 being still able to send the speech response.
Mind I don't really know if you can send two concatenated speech requests, I'm spitballin', know nothing about Alexa skills, just got it yesterday.

**Comment by LorenzHW** [comment]
Yes it probably needs to be truncated to 8000 chars then. Are these logs from Amazon Cloud Watch?

**Comment by Dil3mm4** [comment]
>Yes it probably needs to be truncated to 8000 chars then. Are these logs from Amazon Cloud Watch?

No, those have been obtained by Alexa web app in the home section.
